commands/microservice

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+export LC_ALL=C.UTF-8
+help() {
+  cat <<EOF
+
+----------------------------------------------------------------------------------------------------------
+HELP MESSAGE:
+    Usage: c8y bpl $(basename "$0") <commands> --id <microservice-id>
+    Commands:
+      subscribe           Subscribe tenant to microservice
+      unsubscribe         Unsubscribe tenant to microservice
+      help                Show this help message and exit
+    Arguments:
+      -i, --id      ID    The ID of the application/microservice (required)
```

Comment:
I'm missing the "--failIfAlreadyDone" and "--insecure" options, but maybe we should add this in a separate PR